package/Dockerfile

```diff
@@ -2,21 +2,15 @@ FROM ubuntu:18.04
 
 WORKDIR /var/submariner
 
-COPY strongswan*.tar.gz /var/tmp
-
-ENV DEBIAN_FRONTEND=noninteractive
-
 RUN apt-get -y update && \
-    apt-get -y install curl iproute2 iptables libatm1 libgmp10
-
-RUN /bin/bash -c "cd / && tar xf /var/tmp/strongswan*.tar.gz && rm -f /var/tmp/strongswan*.tar.gz"
-
-COPY charon.conf /usr/local/etc/strongswan.d/charon.conf
-COPY submariner.sh /usr/local/bin
-
-RUN chmod +x /usr/local/bin/submariner.sh
```

Comment:
The first issue i saw is that entrypoint submariner was not executable. After fixing that we get an ipsec related error
Failed to connect to charon: dial unix /var/run/charon.vici: connect: no such file or directory.

Reply:
> The first issue i saw is that entrypoint submariner.sh was not executable. After fixing that we get an ipsec related error
> Failed to connect to charon: dial unix /var/run/charon.vici: connect: no such file or directory.

The commit updates the permissions on `submariner.sh` so that it is executable ;-). I get the `charon.vici` error immediately, I’m looking into what’s wrong in `charon`’s configuration.

Reply:
> The commit updates the permissions on `submariner.sh` so that it is executable ;-). I get the `charon.vici` error immediately, I’m looking into what’s wrong in `charon`’s configuration.

Adding `strongswan-swanctl` allows the e2e tests to pass.